### PR TITLE
Clean up archive.html and archive.js

### PR DESCRIPTION
**Pull request by 7malikk**
Code cleanup in terms of indentation, link, script, and meta spacing and ensuring no code break and maintained functionality.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [x] code is in a uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [x] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by segun-codes** [comment]
Great effort @7malikk.

**Comment by 7malikk** [comment]
> Great effort @7malikk.

Thanks @segun-codes

File: examples/js/archive.js [hunk 1]
```diff
@@ -1,125 +1,124 @@
...
+  axios.get(url)
       .then((response) => {
         if (response.data.files && response.data.files.length != 0) {
           let imageCount = 0;
-          response.data.files.forEach(file => {
-            if (file.format === 'PNG' || file.format === 'JPEG'){
-              let imageRow = document.createElement('div');
-              let image = new Image(150, 150);
-              let placeButton = document.createElement('a');
+          response.data.files.forEach((file) => {
```
Comment (jywarren) [inline examples/js/archive.js:69]: This looks great. I'm going to merge it -- maybe a next step could be to break out lines 70-96 in a separate function called, what do you think... `renderImages`? or would it be better to have separate ones like `renderPng()` and `renderJpeg()`? Yeah, maybe that sounds better?

**Review by jywarren — APPROVED** [review]

File: examples/js/archive.js (hunk 1 above)
Comment (7malikk) [reply examples/js/archive.js:69]: Oh, that sounds a lot better @jywarren 
I'd add that next. Thanks for the review

Comment (7malikk) [reply examples/js/archive.js:69]: I was just going through this again, i think `renderImages`  would be better as there is no distinction when we render a .png or .jpeg file as noticed here:

https://github.com/7malikk/Leaflet.DistortableImage/blob/b396b5a0023183605fa4354626c886638659bf34/examples/js/archive.js#L69-L92

There is no different render method depending on file type.

What do you think @jywarren?

**Comment by jywarren** [comment]
that sounds good! Thanks!

**Pull request merged**